### ISSUE TITLE
SMTP: client.sendMessage(builder, true) not working as expected

### ISSUE THREAD

**Issue by ali7234** [Enough-Software/enough_mail#27]
sendMessage function not return any response, also email not sent to the recipient.

**Code**
```
var client = SmtpClient('enough.de', isLogEnabled: true);
    await client.connectToServer('smtp.domain.com', 465, isSecure: true);
    var loginResponse = await client.login(username, password);
    if (loginResponse.isOkStatus) {
      var builder = MessageBuilder.buildSimpleTextMessage(MailAddress(sender, from), [MailAddress(recipient, to)], composeEmail, subject: subject);
      var sendResponse = await client.sendMessage(builder, false);
      print('message sent: ${sendResponse.message}');
}
```

**Comment by robert-virkus** [comment]
Thanks for your bug report, Ali!
Can you please provide the log output of sending the message? The relevant log section should start with a `MAIL FROM:<senderaddress>`.

**Comment by MBilalMurtaZa** [comment]
Thanks ALi for reporting issue.
I ma facing the same issue in my case(login response is true)
>>>>>>>my code is here.

var client = SmtpClient('xxxxxx-hosting.com, isLogEnabled: true);
    await client.connectToServer('xxxxxx-hosting.com', 465, isSecure: true);
//    await client.startTls();
    var server = await client.ehlo();
    var loginResponse = await client.login(xxx@xxxx.co.uk', 'xxxxxxx');
    print('login response is ${loginResponse.isOkStatus}');
    if (loginResponse.isOkStatus) {
      var builder = MessageBuilder.prepareMultipartAlternativeMessage();
      builder.from = [MailAddress('Bilal', 'xxx@xxxx.co.uk')];
      builder.to = [MailAddress('Ali', 'xxx@xxxx.co.uk')];
      builder.subject = 'My first message';
      builder.addPlainText('hello world.');
      builder.addHtmlText('<p>hello <b>world</b></p>');
      var mimeMessage = builder.buildMimeMessage();
      var sendResponse = await client.sendMessage(mimeMessage);
      print('message sent: ${sendResponse.isOkStatus}');



>>>>>here is my log 
flutter: connecting to server xxxxxx-hosting.com:465 - secure: true
flutter: S: 220-pxxxxxx-hosting.com ESMTP Exim 4.93 #2 Wed, 15 Apr 2020 06:30:49 -0400
flutter: S: 220-We do not authorize the use of this system to transport unsolicited,
flutter: S: 220 and/or bulk e-mail.
flutter: S:
flutter: C: EHLO xxxxxx-hosting.com
flutter: S: 250-xxxxxx-hosting.com Hello xxxxxx-hosting.com [39.53.80.42]
flutter: S: 250-SIZE 52428800
flutter: S: 250-8BITMIME
flutter: S: 250-PIPELINING
flutter: S: 250-AUTH PLAIN LOGIN
flutter: S: 250 HELP
flutter: S:
flutter: C: AUTH PLAIN <base64 code scrambled>
flutter: S: 235 Authentication succeeded
flutter: S:
flutter: login response is true
flutter: C: MAIL FROM:<xxx@xxxx.co.uk> BODY=8BITMIME
flutter: S: 250 OK
flutter: S:

**Comment by ali7234** [comment]
**LOG**
```
I/flutter ( 7440): C: MAIL FROM:<sender@abc.com> BODY=8BITMIME
I/flutter ( 7440): S: 250 OK
I/flutter ( 7440): S:
```

**Comment by robert-virkus** [comment]
Many thanks for the quick feedback, stupid error in the recipients handling. This is now fixed with release v 0.0.10
